src/main/kotlin/no/nav/syfo/behandlendeenhet/domain/BehandlendeEnhet.kt

```diff
@@ -0,0 +1,6 @@
+package no.nav.syfo.behandlendeenhet.domain
+
+data class BehandlendeEnhet(
```

Comment:
Kunne noe slikt vært med å tydeliggjøre hva denne data classen egentlig representerer? 😊 Gjerne annet innhold altså, dette er bare et forslag.
```suggestion
/**
 * Representerer behandlende enhet for en sykmeldt
 *
 * @property geografiskEnhet Den geografiske enheten til den sykmeldte. Grunnregelen er at den sykmeldte får oppfølging her.
 * @property oppfolgingsenhet Om oppfølgingen av en eller annen grunn ikke skal skje ved den geografiske enheten er det oppgitt en oppfølgingsenhet.
 */
data class BehandlendeEnhet(
```